src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, changeQuery] = useState('');
+  const visibleMovies = moviesFromServer
+    .filter(movie => (
+      movie.description.toLowerCase().includes(query.toLowerCase())
```

Comment:
split this with variable
also, DRY, make a separate function for check if `str` includes `substr`
Also, you can make formation in this fucntion